### PR TITLE
initial version of SystemJS/TypeScript sample

### PR DESCRIPTION
**Pull request by vladima**
This PR should be updated after TypeScript 1.5 is released and [PR to ES6 module loader](https://github.com/ModuleLoader/es6-module-loader/pull/366) is accepted

### CODE REVIEW

**Comment by vladima** [comment]
pringing @mhegazy

**Comment by mhegazy** [comment]
:+1:

File: systemjs/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+**Fetch dependencies:**
+```
+npm install
+```
+
+**Run (using port 8080)**
+```
+npm start
```
Comment (mhegazy) [inline systemjs/README.md]: why do you need this?

File: systemjs/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+```
+
+**Run (using port 8080)**
+```
+npm start
+```
+
+**Run server on custom port**
+```
+node node_modules/http-server/bin/http-server -p 8080
```
Comment (mhegazy) [inline systemjs/README.md]: possibly add a -o to open the browser window

**Pull request merged**